inc/css/class-block-css.php

```diff
@@ -0,0 +1,322 @@
+<?php
+
+namespace ThemeIsle;
+
+if ( ! class_exists( '\ThemeIsle\BlockCSS' ) ) {
+	/**
+	 * Class BlockCSS
+	 *
+	 * @package BlockCSS
+	 */
+	class BlockCSS {
+
+		/**
+		 * The namespace under which the blocks are registered.
+		 *
+		 * @var string
+		 */
+		protected $library_prefix = 'themeisle-blocks';
+
+		/**
+		 * The namespace under which the block classees are saved.
+		 *
+		 * @var string
+		 */
+		protected static $blocks_classes = array();
+
+		/**
+		 * The namespace under which the fonts are saved.
+		 *
+		 * @var array
+		 */
+		protected static $google_fonts = array();
+
+		/**
+		 * BlockCSS constructor.
+		 *
+		 * @since   1.2.5
+		 * @access  public
+		 */
+		public function __construct() {
+			add_action( 'init', array( $this, 'autoload_block_classes' ), 99 );
+			add_filter( 'safe_style_css', array( $this, 'used_css_properties' ), 99 );
+		}
+
+		/**
+		 * Autoload classes for each block.
+		 *
+		 * @since   1.2.5
+		 * @access  public
+		 */
+		public function autoload_block_classes() {
+			$paths = glob( $this->get_dir() . '/../editor/*/*/*-css.php' );
+			foreach ( $paths as $path ) {
+				require_once $path;
+				// remove the class prefix and the extension
+				$classname = str_replace( array( 'class-', '.php' ), '', basename( $path ) );
+				// get an array of words from class names and we'll make them capitalized.
+				$classname = explode( '-', $classname );
+				$classname = array_map( 'ucfirst', $classname );
+				// rebuild the classname string as capitalized and separated by underscores.
+				$classname = implode( '_', $classname );
+				$classname = str_replace( 'Css', 'CSS', $classname );
+				$classname = 'ThemeIsle\BlockCSS\\' . $classname;
+
+				if ( ! class_exists( $classname ) ) {
+					continue;
+				}
+
+				self::$blocks_classes[] = $classname;
+			}
+		}
+
+		/**
+		 * Parse Blocks for Gutenberg and WordPress 5.0
+		 * 
+		 * @since   1.2.5
+		 * @access  public
+		 */
+		public function parse_blocks( $content ) {
+			if ( ! function_exists( 'parse_blocks' ) ) {
+				return gutenberg_parse_blocks( $content );
+			} else {
+				return parse_blocks( $content );
+			}
+		}
+
+		/**
+		 * Get block attribute value with default
+		 * 
+		 * @since   1.2.5
+		 * @access  public
+		 */
+		public function get_attr_value( $attr, $default = 'unset' ) {
+			if ( isset( $attr ) ) {
```

Comment:
this seems not right. $atts will be always defined and `isset()` is checking if the variable is defined. I think you wanted to use empty?

Reply:
Fixed https://github.com/Codeinwp/gutenberg-blocks/pull/119/commits/0488851e4bd051c0f90104b990ef0f226e13f4a5